src/webview/wkwebview/mod.rs

```diff
@@ -158,6 +163,8 @@ impl InnerWebView {
             let _: () = msg_send![body_stream, close];
           }
 
+          tracing::debug!("done reading body");
```

Comment:
is this a leftover? seems like other platforms don't have a similar `debug!` call